manifests/storage_state_crd.yaml

```diff
@@ -17,3 +17,63 @@ spec:
   - name: v1alpha1
     served: true
     storage: true
+  "validation":
+    "openAPIV3Schema":
```

Comment:
Do you know why these two lines are quoted?

Reply:
Some encoding issue of the yaml library. We saw that before with controller-gen. Maybe @directxman12 has an idea.